rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2279,3 +2279,153 @@ rocalToDecibels(
+}
+
+RocalTensor rocalTensorMulScalar(RocalContext p_context,
+                                 RocalTensor p_input,
+                                 bool is_output,
+                                 RocalTensorOutputType rocal_tensor_output_type,
+                                 float scalar) {
+    if (!p_context || !p_input)
+        THROW("Null values passed as input")
+    Tensor* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    RocalTensorDataType op_tensor_data_type;
+    try {
+        op_tensor_data_type = (RocalTensorDataType)rocal_tensor_output_type;
+        TensorInfo output_info = input->info();
+        output_info.set_data_type(op_tensor_data_type);
+        output = context->master_graph->create_tensor(output_info, is_output);
+        context->master_graph->add_node<TensorMulScalarNode>({input}, {output})->init(scalar);
+    } catch (const std::exception& e) {
+        context->capture_error(e.what());
+        ERR(e.what())
+    }
+    return output;
+}
+
+RocalTensor rocalTensorAddTensor(RocalContext p_context,
+                                 RocalTensor p_input1,
+                                 RocalTensor p_input2,
+                                 bool is_output,
+                                 RocalTensorOutputType rocal_tensor_output_datatype) {
+    if (!p_context || !p_input1 || !p_input2)
+        THROW("Null values passed as input")
+    Tensor* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input1 = static_cast<Tensor*>(p_input1);
+    auto input2 = static_cast<Tensor*>(p_input2);
+    RocalTensorDataType op_tensor_data_type;
+    try {
+        op_tensor_data_type = (RocalTensorDataType)rocal_tensor_output_datatype;
+        TensorInfo output_info = input1->info();
+        output_info.set_data_type(op_tensor_data_type);
+        output = context->master_graph->create_tensor(output_info, is_output);
+        context->master_graph->add_node<TensorAddTensorNode>({input1, input2}, {output})->init();
+    } catch (const std::exception& e) {
+        context->capture_error(e.what());
+        ERR(e.what())
+    }
+    return output;
+}
+
+RocalTensor rocalUniformDistribution(RocalContext p_context,
+                                     RocalTensor p_input,
+                                     bool is_output,
+                                     std::vector<float>& range) {
+    if (!p_context)
+        THROW("Null values passed as input")
+    Tensor* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    try {
+        std::cerr << "Here in Uniform Dist";
+        RocalTensorDataType tensor_data_type = RocalTensorDataType::FP32;
+        unsigned num_of_dims = 3;
+        std::vector<size_t> dims;
+        dims.resize(num_of_dims);
+        dims.at(0) = context->user_batch_size();
+        dims.at(1) = 1;
+        dims.at(2) = 1;
```

Comment:
```suggestion
        unsigned num_of_dims = 3;
        std::vector<size_t> dims;
        dims.resize(num_of_dims);
        dims.at(0) = context->user_batch_size();
        dims.at(1) = 1;
        dims.at(2) = 1;
```

Add all this in a single line
`std::vector<size_t> dims = {context->user_batch_size(), 1, 1}`